.github/workflows/build.yml

```diff
@@ -68,3 +75,6 @@ jobs:
 
     - name: Run NUKE
       run: ./build.sh UnitTests
+    
+    - name: Unix spell check tests
+      run: ./build.sh spellcheck
```

Comment:
I added this only for testing purposes.. Will remove this before merge :)

Reply:
Yes, please remove it.